samples/Audio/SimpleAudio/SimpleAudio.Windows/SimpleAudio.Windows.csproj

```diff
@@ -1,58 +1,17 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net8.0-windows7.0</TargetFramework>
+    <TargetFramework>net8.0-windows</TargetFramework>
     <ApplicationIcon>Resources\GameIcon.ico</ApplicationIcon>
     <OutputType>WinExe</OutputType>
     <RootNamespace>SimpleAudio</RootNamespace>
     <OutputPath>..\Bin\Windows\$(Configuration)\</OutputPath>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <DefineConstants>STRIDE_PLATFORM_DESKTOP</DefineConstants>
```

Comment:
It looks like the `TRACE` constants was also defined in all configuration. Not sure what it is used for though.

Reply:
It seems to be useful when we want to use the Trace class and/or print out trace logs. I don't think it is a useful default for game samples. So I'm ok with leaving it out.

Reference: https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.trace